documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -67,11 +79,31 @@ data:
   user.password: # Password for protecting the PKCS #12 archive file
 ----
 
-== SCRAM-SHA-512 Authentication
+== TLS client authentication using an external certificate
+
+To use TLS client authentication using a certificate issued outside the User Operator, you set the `type` field in the `KafkaUser` resource to `tls-external`.
+A secret and credentials are not created for the user.
+
+.Example user with TLS client authentication using external certificates enabled
```

Comment:
Same as above.

Reply:
Changed to: _.Example user with TLS client authentication that uses a certificate issued outside the User Operator_